### ISSUE TITLE
Convert the list to a table?

### ISSUE THREAD

**Issue by roke-julian-lockhart** [GHolk/addy-io-unofficial#4]
#### List

Consider [our current list](https://github.com/GHolk/addy-io-unofficial/wiki/Home/83e6d4526eed797137ab34193211748e9710eed3#list:~:text=Adobe%20Creative%20Cloud:%20prepare.%20Works,Spotify:%20broken.):

<blockquote>

- [Adobe Creative Cloud](https://auth.services.adobe.com/en_GB/index.html#/signup:~:text=1%20of%202-,Create%20an%20account,-Or): **prepare**. Works in the website, but not work with app. [ref](https://www.reddit.com/r/AnonAddy/comments/lpi635/services_known_to_block_anonaddy_default_domains/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button#:~:text=Adobe%20Cloud%20(logging%20into%20their%20website%20works%20fine%2C%20but%20not%20into%20any%20of%20their%20smartphone%20apps))
- [character.ai](https://character.ai/): **broken**.
- [Discord](https://discord.com/register): **broken** **allow-plus** **allow-dot**.

- [Facebook](https://www.facebook.com/r.php?locale=en_GB): **prepare** **allow-plus**. Changing the e-mail to an Addy one after registration works:

   1. Change the email to receive ad to an AnonAddy mail.
   2. Then, we can receive the ad verification mail.
   3. Finally, change the contact mail to AnonAddy, and we can receive the verification mail. [ref](https://www.reddit.com/r/AnonAddy/comments/lpi635/comment/iu8nyaq/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button#:~:text=Ok%2C%20I%20got%20facebook%20accept%20anonaddy.%20First%2C%20I%20change%20the%20email%20to%20receive%20ad%20to%20an%20anonaddy%20mail.%20Then%2C%20I%20receive%20the%20ad%20verification%20mail.%20Finally%2C%20change%20the%20contact%20mail%20to%20anonaddy%2C%20and%20I%20can%20receive%20the%20verification%20mail.)
	 
- [OpenAI.com](https://platform.openai.com/signup): **broken**.
- [Pinterest](https://pinterest.com/): **broken**.
- [Spotify](https://www.spotify.com/uk/signup): **broken**.

</blockquote>

<details>

```Markdown
- [Adobe Creative Cloud](https://auth.services.adobe.com/en_GB/index.html#/signup:~:text=1%20of%202-,Create%20an%20account,-Or): **prepare**. Works in the website, but not work with app. [ref](https://www.reddit.com/r/AnonAddy/comments/lpi635/services_known_to_block_anonaddy_default_domains/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button#:~:text=Adobe%20Cloud%20(logging%20into%20their%20website%20works%20fine%2C%20but%20not%20into%20any%20of%20their%20smartphone%20apps))
- [character.ai](https://character.ai/): **broken**.
- [Discord](https://discord.com/register): **broken** **allow-plus** **allow-dot**.

- [Facebook](https://www.facebook.com/r.php?locale=en_GB): **prepare** **allow-plus**. Changing the e-mail to an Addy one after registration works:

   1. Change the email to receive ad to an AnonAddy mail.
   2. Then, we can receive the ad verification mail.
   3. Finally, change the contact mail to AnonAddy, and we can receive the verification mail. [ref](https://www.reddit.com/r/AnonAddy/comments/lpi635/comment/iu8nyaq/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button#:~:text=Ok%2C%20I%20got%20facebook%20accept%20anonaddy.%20First%2C%20I%20change%20the%20email%20to%20receive%20ad%20to%20an%20anonaddy%20mail.%20Then%2C%20I%20receive%20the%20ad%20verification%20mail.%20Finally%2C%20change%20the%20contact%20mail%20to%20anonaddy%2C%20and%20I%20can%20receive%20the%20verification%20mail.)
	 
- [OpenAI.com](https://platform.openai.com/signup): **broken**.
- [Pinterest](https://pinterest.com/): **broken**.
- [Spotify](https://www.spotify.com/uk/signup): **broken**.
```

</details>

I don't consider this to be trivial to understand, especially when skimming it.

#### Table

Instead, I propose the undermentioned:

<table>
	<thead>
		<tr>
			<th>
				Site
			</th>
			<th>
				Tags
			</th>
			<th>
				Description
			</th>
		</tr>
	<tbody>
		<tr>
			<td>
				<a href="https://auth.services.adobe.com/en_GB/index.html#/signup:~:text=1%20of%202-,Create%20an%20account,-Or">Adobe Creative Cloud</a>
			</td>
			<td>
				Prepare
			</td>
			<td>
				Works in the website, but does not work with the app. <a href="https://www.reddit.com/r/AnonAddy/comments/lpi635/services_known_to_block_anonaddy_default_domains/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button#:~:text=Adobe%20Cloud%20(logging%20into%20their%20website%20works%20fine%2C%20but%20not%20into%20any%20of%20their%20smartphone%20apps)">Reference</a>
			</td>
		</tr>
		<tr>
			<td>
				<a href="https://character.ai/"><code>character.ai</code></a>
			</td>
			<td>
				Broken
			</td>
			<td>
			</td>
		</tr>
		<tr>
			<td>
				<a href="https://discord.com/register">Discord<a>
			</td>
			<td>
				<ol>
					<li>
						Broken
					</li>
					<li>
						Allow-plus
					</li>
					<li>
						Allow-dot
					</li>
				</ol>
			</td>
			<td>
			</td>
		</tr>
		<tr>
			<td>
				<a href="https://www.facebook.com/r.php?locale=en_GB">Facebook<a>
			</td>
			<td>
				<ol>
					<li>
						Prepare
					</li>
					<li>
						Allow-plus
					</li>
				</ol>
			</td>
			<td>
				<p>
					Changing the e-mail to an Addy one after registration works:
				</p>
				<ol>
					<li>
						Change the email to receive ad to an AnonAddy mail.
					</li>
					<li>
						Then, we can receive the ad verification mail.
					</li>
					<li>
						Finally, change the contact mail to AnonAddy, and we can receive the verification mail. <a href="https://www.reddit.com/r/AnonAddy/comments/lpi635/comment/iu8nyaq/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button#:~:text=Ok%2C%20I%20got%20facebook%20accept%20anonaddy.%20First%2C%20I%20change%20the%20email%20to%20receive%20ad%20to%20an%20anonaddy%20mail.%20Then%2C%20I%20receive%20the%20ad%20verification%20mail.%20Finally%2C%20change%20the%20contact%20mail%20to%20anonaddy%2C%20and%20I%20can%20receive%20the%20verification%20mail.">Reference</a>
					</li>
				</ol>
			</td>
		</tr>
		<tr>
			<td>
				<a href="https://platform.openai.com/signup"><code>OpenAI.com</code></a>
			</td>
			<td>
				Broken
			</td>
			<td>
			</td>
		</tr>
		<tr>
			<td>
				<a href="https://pinterest.com/">Pinterest</a>
			</td>
			<td>
				Broken
			</td>
			<td>
			</td>
		</tr>
		<tr>
			<td>
				<a href="https://www.spotify.com/uk/signup">Spotify</a>
			</td>
			<td>
				Broken
			</td>
			<td>
			</td>
		</tr>
	</tbody>
</table>

<details>

```Markdown
<table>
	<thead>
		<tr>
			<th>
				Site
			</th>
			<th>
				Tags
			</th>
			<th>
				Description
			</th>
		</tr>
	<tbody>
		<tr>
			<td>
				<a href="https://auth.services.adobe.com/en_GB/index.html#/signup:~:text=1%20of%202-,Create%20an%20account,-Or">Adobe Creative Cloud</a>
			</td>
			<td>
				Prepare
			</td>
			<td>
				Works in the website, but does not work with the app. <a href="https://www.reddit.com/r/AnonAddy/comments/lpi635/services_known_to_block_anonaddy_default_domains/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button#:~:text=Adobe%20Cloud%20(logging%20into%20their%20website%20works%20fine%2C%20but%20not%20into%20any%20of%20their%20smartphone%20apps)">Reference</a>
			</td>
		</tr>
		<tr>
			<td>
				<a href="https://character.ai/"><code>character.ai</code></a>
			</td>
			<td>
				Broken
			</td>
			<td>
			</td>
		</tr>
		<tr>
			<td>
				<a href="https://discord.com/register">Discord<a>
			</td>
			<td>
				<ol>
					<li>
						Broken
					</li>
					<li>
						Allow-plus
					</li>
					<li>
						Allow-dot
					</li>
				</ol>
			</td>
			<td>
			</td>
		</tr>
		<tr>
			<td>
				<a href="https://www.facebook.com/r.php?locale=en_GB">Facebook<a>
			</td>
			<td>
				<ol>
					<li>
						Prepare
					</li>
					<li>
						Allow-plus
					</li>
				</ol>
			</td>
			<td>
				<p>
					Changing the e-mail to an Addy one after registration works:
				</p>
				<ol>
					<li>
						Change the email to receive ad to an AnonAddy mail.
					</li>
					<li>
						Then, we can receive the ad verification mail.
					</li>
					<li>
						Finally, change the contact mail to AnonAddy, and we can receive the verification mail. <a href="https://www.reddit.com/r/AnonAddy/comments/lpi635/comment/iu8nyaq/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button#:~:text=Ok%2C%20I%20got%20facebook%20accept%20anonaddy.%20First%2C%20I%20change%20the%20email%20to%20receive%20ad%20to%20an%20anonaddy%20mail.%20Then%2C%20I%20receive%20the%20ad%20verification%20mail.%20Finally%2C%20change%20the%20contact%20mail%20to%20anonaddy%2C%20and%20I%20can%20receive%20the%20verification%20mail.">Reference</a>
					</li>
				</ol>
			</td>
		</tr>
		<tr>
			<td>
				<a href="https://platform.openai.com/signup"><code>OpenAI.com</code></a>
			</td>
			<td>
				Broken
			</td>
			<td>
			</td>
		</tr>
		<tr>
			<td>
				<a href="https://pinterest.com/">Pinterest</a>
			</td>
			<td>
				Broken
			</td>
			<td>
			</td>
		</tr>
		<tr>
			<td>
				<a href="https://www.spotify.com/uk/signup">Spotify</a>
			</td>
			<td>
				Broken
			</td>
			<td>
			</td>
		</tr>
	</tbody>
</table>
```

</details>

I think it's easier to visually parse.

**Comment by GHolk** [comment]
Personally I don't like table.
It is hard to parse as text (markdown) or html,
and it does not render will on smartphones and the small screen.

To anyone:
If you perfer table to list, comment or react :+1: on the origin author.

**Comment by roke-julian-lockhart** [comment]
> [...] it does not render will on smartphones and the small screen.

@GHolk, do you consider [the undermentioned](https://chatgpt.com/share/672501b4-4e30-800f-868b-3f7ce13a09d6) to be an improvement? It's much more of a combination of your proposed list, whilst also making each entry easier to read.

1. <table>
      <tbody>
         <tr>
            <th>Site</th>
            <td><a href="https://auth.services.adobe.com/en_GB/index.html#/signup:~:text=1%20of%202-,Create%20an%20account,-Or">Adobe Creative Cloud</a></td>
         </tr>
         <tr>
            <th>Tags</th>
            <td>Prepare</td>
         </tr>
         <tr>
            <th>Description</th>
            <td>Works in the website, but does not work with the app. <a href="https://www.reddit.com/r/AnonAddy/comments/lpi635/services_known_to_block_anonaddy_default_domains/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button#:~:text=Adobe%20Cloud%20(logging%20into%20their%20website%20works%20fine%2C%20but%20not%20into%20any%20of%20their%20smartphone%20apps)">Reference</a></td>
         </tr>
      </tbody>
   </table>

2. <table>
      <tbody>
         <tr>
            <th>Site</th>
            <td><a href="https://character.ai/"><code>character.ai</code></a></td>
         </tr>
         <tr>
            <th>Tags</th>
            <td>Broken</td>
         </tr>
      </tbody>
   </table>

3. <table>
      <tbody>
         <tr>
            <th>Site</th>
            <td><a href="https://discord.com/register">Discord</a></td>
         </tr>
         <tr>
            <th>Tags</th>
            <td>
               <ul>
                  <li>Broken</li>
                  <li>Allow-plus</li>
                  <li>Allow-dot</li>
               </ul>
            </td>
         </tr>
      </tbody>
   </table>

4. <table>
      <tbody>
         <tr>
            <th>Site</th>
            <td><a href="https://www.facebook.com/r.php?locale=en_GB">Facebook</a></td>
         </tr>
         <tr>
            <th>Tags</th>
            <td>
               <ul>
                  <li>Prepare</li>
                  <li>Allow-plus</li>
               </ul>
            </td>
         </tr>
         <tr>
            <th>Description</th>
            <td>
               <p>Changing the e-mail to an Addy one after registration works:</p>
               <ol>
                  <li>Change the email to receive ad to an AnonAddy mail.</li>
                  <li>Then, we can receive the ad verification mail.</li>
                  <li>Finally, change the contact mail to AnonAddy, and we can receive the verification mail. <a href="https://www.reddit.com/r/AnonAddy/comments/lpi635/comment/iu8nyaq/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button#:~:text=Ok%2C%20I%20got%20facebook%20accept%20anonaddy.%20First%2C%20I%20change%20the%20email%20to%20receive%20ad%20to%20an%20anonaddy%20mail.%20Then%2C%20I%20receive%20the%20ad%20verification%20mail.%20Finally%2C%20change%20the%20contact%20mail%20to%20anonaddy%2C%20and%20I%20can%20receive%20the%20verification%20mail.">Reference</a></li>
               </ol>
            </td>
         </tr>
      </tbody>
   </table>

In retrospect, I prefer the formatting of this version, even if it is less efficient. I could convert the "Site" sections to headings instead of table entries themselves, too.

**Comment by GHolk** [comment]
I would prefer to write a script to convert the simple list to some big table(s),
but not hand craft the tables in markdown.

**Comment by roke-julian-lockhart** [comment]
> https://github.com/GHolk/addy-io-unofficial/issues/4#issuecomment-2452220942

@GHolk, I understand that. This would be a lot easier if GitHub supported https://github.com/11ty/eleventy/issues/2438#issue-1271419451, since I wouldn't need to delegate to HTML except for the basic table structure itself.

We could just use headings instead, but that would take a lot of vertical space. It would allow readers to link to specific subsections (like the descriptions) though.

**Comment by GHolk** [comment]
About link, my idea is to split list and add alphabate headings when the list is too large, like `### A-E` or `### A`.

**Comment by roke-julian-lockhart** [comment]
> https://github.com/GHolk/addy-io-unofficial/issues/4#issuecomment-2452835983

@GHolk, a good idea – I agree. However, it doesn't help much for parsing the different aspects of each list entry, which matters more right now, considering that we only have a few entries on the list.

**Comment by bronze6714** [comment]
yo guys i really don't know what you mean by broken but its working for spotify for me...for like 1-2 months now

**Comment by GHolk** [comment]
@bronze6714 you mean that we can register on spotify with addy.io's mail address now?
I have updated the wiki.

**Comment by roke-julian-lockhart** [comment]
> https://github.com/GHolk/addy-io-unofficial/issues/4#issuecomment-2561900323

@GHolk, yeah. I did it last year.
